src/main/java/org/jabref/model/entry/LinkedFile.java

```diff
@@ -33,7 +33,14 @@
 
     public LinkedFile(String description, String link, String fileType) {
         this.description.setValue(Objects.requireNonNull(description));
-        this.link.setValue(Objects.requireNonNull(link));
+
+        String fileLink = Objects.requireNonNull(link);
```

Comment:
What is the point of adding the additional variable named `fileLink` here? Since Strings are immutable you can just keep using `link` directly below.

Reply:
Or even shorter: `setLink(Objects.requireNonNull(link))`, which also removes a bit of code duplication.